tests/datasource/dev_get_prev_partition.py

```diff
@@ -1,118 +0,0 @@
-import os
-import shutil
-from typing import List, Union
-
-import pandas as pd
```

Comment:
This was an old file that I created while doing batchpod work. Removed with this PR.